android/java/org/chromium/chrome/browser/preferences/BravePreferenceFragment.java

```diff
@@ -0,0 +1,89 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.preferences;
+
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager.NameNotFoundException;
+import android.preference.Preference;
+import android.preference.PreferenceFragment;
+import java.util.HashMap;
+import org.chromium.base.Log;
+import org.chromium.chrome.browser.ChromeVersionInfo;
+import org.chromium.chrome.browser.preferences.PrefServiceBridge.AboutVersionStrings;
+import org.chromium.chrome.browser.preferences.website.SiteSettingsCategory;
+import org.chromium.chrome.browser.preferences.website.SiteSettingsPreferences;
+
+/**
+ * Base class for preferences.
+ */
+public class BravePreferenceFragment extends PreferenceFragment {
+    private static final String PREF_SAFE_BROWSING = "safe_browsing";
+    private static final String PREF_DO_NOT_TRACK = "do_not_track";
+    private static final String PREF_USAGE_AND_CRASH_REPORTING = "usage_and_crash_reports";
```

Comment:
why is this disabled? We have crash reporting in desktop, but we default to disabled. I think that's the behavior we want here too?

Reply:
We don't have this option currently on Android. @bbondy @SergeyZhukovsky do we need it on `brave-core`'s Android?